### ISSUE TITLE
VMware and Ansible effectively dead?

### ISSUE THREAD

**Issue by brosenqvist** [vmware/ansible-for-nsxt#521]
### Is your feature request related to a problem? Please describe.

I assume by the looks of this repo, that VMware has basically dropped all support for Ansible?

### Describe the solution you'd like

Better support for ansible?

**Comment by fs30000** [comment]
Yeah, it's dead, still for what i am doing it still works. Anything that a specific module doesn't do, you can always do it a more manual way by using the generic nsxt_rest.  Check the API here: https://developer.broadcom.com/xapis/nsx-t-data-center-rest-api/4.0.1/#api
